ONSdigital/florence
# Allow user to retry file upload on failure

### What

Allow a user to retry a file uploaded if there has been an error during the initial upload process.

### How to review

You'll need Florence on the `feature/resume-dataset-upload` branch, as well as the AWS access ID and keys exported to your console. You'll also need the recipe and import API running. Follow these steps:

1) Navigate to `/florence/datasets`
2) Click 'Upload new version' of the 'CPI COICOP' dataset
3) Switch your network off (to force an upload error)
4) Attempt to upload a file (you should be given an error)
5) Click retry, switch your network back on and check that the file uploads fine.

### Who can review

Anyone but me.
